pages/home.html

```diff
@@ -3,6 +3,7 @@
 description: The United States Digital Service is transforming how the federal government works for the American people. And we need you.
 permalink: /
 social_image: social-card--homepage.jpg
+social_image_alt_text: "A dozen USDS staff members are seated, using laptops around a presentation white board that illustrates a workflow diagram."
```

Comment:
Suggested text edit:

"A group of USDS staff members are seated on chairs and couches in the organization's headquarters. They are working on laptops and listening to a presentation from a colleague, who is pointing to a workflow diagram drawn on a white board."